tasking/pulp/tasking/celery_app.py

```diff
@@ -69,6 +69,8 @@ def initialize_worker(sender, instance, **kwargs):
     :param kwargs:   Other params (unused)
     :type  kwargs:   dict
     """
+    from pulp.tasking.services.worker_watcher import delete_worker
```

Comment:
Is it worth creating a refactor task in redmine? (I can do it if needed) Or we are going to look for a better approach only in case of some problems so it will be resolved in some real issue?